build.sbt

```diff
@@ -716,6 +716,20 @@ lazy val cli = project
     Test / parallelExecution := false
   )
 
+lazy val `task-progress-notifications` = project
```

Comment:
You need to add this to the global project.

Reply:
Pheew, thanks for catching that, I don't know why I'm always forgetting this...